dockerfiles/pytorch-dev/pytorch_dev_ubuntu_24.04.Dockerfile

```diff
@@ -44,12 +44,14 @@ RUN git config --global user.email "you@example.com" && \
 # /therock/src
 # workaround is to disable selinux (/etc/selinux/config)
 RUN --mount=type=bind,target=/therock/src \
-  python3 /therock/src/external-builds/pytorch/pytorch_vision_repo.py checkout \
-    --repo /therock/pytorch_vision --depth 1 --jobs 10
+  python3 /therock/src/external-builds/pytorch/ptbuild.py checkout \
+    --no-pytorch --no-audio \
+    --vision-src-dir /therock/pytorch_vision --depth 1 --jobs 10
 # 2. audio
 RUN --mount=type=bind,target=/therock/src \
-  python3 /therock/src/external-builds/pytorch/pytorch_audio_repo.py checkout \
-    --repo /therock/pytorch_audio --depth 1 --jobs 10
+  python3 /therock/src/external-builds/pytorch/ptbuild.py checkout \
+    --no-pytorch --no-vision \
+    --audio-src-dir /therock/pytorch_audio --depth 1 --jobs 10
 COPY external-builds/pytorch/env_init.sh /therock/
 COPY external-builds/pytorch/build_pytorch_torch.sh /therock/
 COPY external-builds/pytorch/build_pytorch_vision.sh /therock/
```

Comment:
Ah, these scripts make assumptions about their path relative to the source folders: https://github.com/ROCm/TheRock/blob/5e6b5da2074313b2e1ae1331c14dad97cc734b65/external-builds/pytorch/build_pytorch_vision.sh#L35

Will need to fix that here.

We could also later build out the `ptbuild.py develop` and `ptbuild.py bdist_wheel` commands so we aren't scattered across script files.